services/app-graphql/src/schema.graphql

```diff
@@ -975,7 +975,7 @@ type ContractFormData {
 }
 
 "Either new capitation rates (NEW) or updates to previously certified capitation rates (AMENDMENT)"
-enum RateType {
+enum RateAmendmentType {
```

Comment:
I changed the name on this because I was seeing types getting confused in my editor when `RateType` was both
 1. the enum for a rate's type  (radio button response)
 2. the complex data object coming back from apollo client representing the entire rate (the full data model). . 
 
Reminder, the word type is appended from our codegen settings. I could have changed the suffix but that felt like more wide-reaching code change then just renaming the rate's type enum to something else.